src/MovieContainer/MovieContainer.js

```diff
@@ -4,19 +4,19 @@ import MiniMovieCards from "../MiniMovieCards/MiniMovieCards";
 import './MovieContainer.css'
 
 const MovieContainer = ({ movieData, getDetails }) => {
-  let mappedMovies = movieData.map((movie) => {
-    return (
-      <MiniMovieCards
-        id={movie.id}
-        title={movie.title}
-        poster={movie.poster_path}
-        rating={movie.average_rating}
-        key={movie.id}
-        getDetails={getDetails}
-      />
-    );
-  });
-  return <div className="MovieContainer">{mappedMovies}</div>;
+    let mappedMovies = movieData.map((movie) => {
+        return (
+            <MiniMovieCards
+                id={movie.id}
```

Comment:
If you didn't want to pass so many props, you could pass the whole `movie` object thru and break it apart in `MiniMovieCards`